lecture_1/homework/app/controllers/application_controller.rb

```diff
@@ -1,2 +1,11 @@
 class ApplicationController < ActionController::API
+  private
+
+  def default_limit
+    params[:limit] = 20 if limit.zero?
```

Comment:
Ja tą metodę widziałbym raczej tak :) Taki refaktor sprawia, że nie trzeba 2 razy wyciągać wartości spod klucz `limit` w `params`, nie ma potrzeby konwersji na integer, nawet wiemy wtedy że jak użytkownik poda limit 0 to dostanie 0 rekordów tak jak chce :eyes: Mimo że brzmi głupio :D Jeśli założymy tak jak jest w kodzie, że jednak jeśli użytkownik poda limit 0 to dostaniemy wszystko, to tutaj warto by wyciągniecie danych `params[:limit]` gdzieś wyżej, do innej metody i zmemoizować 
```ruby
def limit
  params[:limit] = params[:limit] || 20
end
```

Reply:
Poprawione :D